### DISCUSSION TITLE
textarea?

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9524] [General]
Do you have any version of textarea apart og legacy?
ANd what about appearence of text area? As i noticed it did not picktextfield look...
How can i pass appearence

**Comment by waterplea** [comment]
Currently we only have legacy textarea. I see the issue with custom appearance with it, I'll get it fixed in next release.

**Reply by vytautas-pranskunas-** [reply]
could you replie here with the fix after nest release  please?

**Reply by waterplea** [reply]
It's released:
https://taiga-ui.dev/components/textarea#appearance

**Reply by vytautas-pranskunas-** [reply]
thanks.
but did not helped. Here is the code

```
<div>
        <label tuiLabel>{{ 'resident-edit.note' | translate }}</label>
        <tui-textarea
            tuiTextfieldAppearance="parama-textfield"
            formControlName="note"
            [maxLength]="2000"
        ></tui-textarea>
        <tui-error formControlName="note" [error]="[] | paramaTuiFieldError | async" />
    </div>
```

and here is the result
![Screenshot 2024-10-21 at 16 15 13](https://github.com/user-attachments/assets/ccfd3eb8-9765-40af-9a8f-9979cf903c38)


do i do something wrong?

**Reply by waterplea** [reply]
You need to import `TuiTextfieldControllerModule` for the directive to work

**Reply by vytautas-pranskunas-** [reply]
That works. Thanks
